src/components/families/family-form/student-form/StudentForm.tsx

```diff
@@ -16,32 +23,45 @@ import { DynamicField } from "types";
 let CHILD_KEY_COUNTER = 1;
 let GUEST_KEY_COUNTER = 1;
 
-const useStyles = makeStyles(() => ({
+const denseStyles = (isEditing: boolean) => ({
   addButton: {
+    marginBottom: 24,
+  },
+  roleLabel: {
+    fontSize: isEditing ? 14 : 15,
+    width: isEditing ? 112 : "",
+  },
+  rowContainer: {
+    height: 32,
+  },
+});
+
+const useStyles = makeStyles<Theme, Pick<Props, "dense" | "isEditing">>(() => ({
+  addButton: ({ dense, isEditing }) => ({
+    marginBottom: 48,
     marginLeft: 32,
     marginTop: 10,
     paddingLeft: 12,
     paddingRight: 12,
-  },
+    ...(dense && denseStyles(isEditing).addButton),
```

Comment:
Question: if dense is False, then the object spread returns null?

Reply:
by logging this object it just doesn't add the extra properties, not sure if there's a better way :o